vdmDriverII.py

```diff
@@ -119,7 +120,7 @@
     table = {}
     csvtable = []
 
-    table, csvtable = doMakeRateFile(makeRateFileConfig)
+    table, csvtable = doMakePCCRateFile(makeRateFileConfig)
```

Comment:
Ah... I see how you made it work.  :)  You can't do this.  Most other luminometers use the regular function.  You can make a if/else statemate:

if datatype=="PCC":
    table, csvtable = doMakePCCRateFile(makeRateFileConfig)
else:
    table, csvtable = doMakeRateFile(makeRateFileConfig)